### ISSUE TITLE
How can we pass the parsed html to html-loader?

### ISSUE THREAD

**Issue by zheeeng** [hmsk/frontmatter-markdown-loader#20]
In some situation, we need some additional works to process on the html content by html-loader

**Comment by hmsk** [comment]
Interesting. Could you give more detail with the actual sample codes? I'd like to think about whether accepting `html-loader` options for `html` property.

As a workaround, you can intercept parsed HTML in `markdown` option by giving markdown compilation manually.
https://github.com/hmsk/frontmatter-markdown-loader#use-your-own-markdown-compile

**Comment by zheeeng** [comment]
We may reference some local assets which need by processed by html-loader.

**Comment by hmsk** [comment]
Understood, I may try to support that by some measures.
It'll be great if you give actual settings for html-loader which you're using.

**Comment by jefrydco** [comment]
I think this issue should be closed @hmsk regarding of merged https://github.com/hmsk/frontmatter-markdown-loader/pull/74

**Comment by hmsk** [comment]
No, this is not fully fulfilled. Imported `html` doesn't work as intentional.
#74 is just for imported `vue` contents.
